pyproject.toml

```diff
@@ -0,0 +1,101 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "bertopic"
+dynamic = ["version"]
```

Comment:
I would prefer an explicit version akin to:

```
version = "0.16.2"
```

Reply:
You want to track the version information only in the pyproject.toml then or duplicate it in the src as well? I recommend keeping a single source of truth.

If the version is set in the toml a common pattern is to use:

```python
from importlib.metadata import version
__version__ = version("bertopic")
```

In case you need the version at runtime.